server/models/cards.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose')
+const cardSchema = new mongoose.Schema({
+    name: {type: String, required: true}, 
+    desc: {type: String, required: true},
+    column: {type: mongoose.Schema.Types.ObjectId, ref: "Column"},
+    deadline: Date,
```

Comment:
Notes/Comment property type is missing

Reply:
The ticket says that I don't have to implement those yet. I also removed the deadline portion as well. I guess we'll be adding those other pieces later.

Reply:
We will be adding those later! This level of detail is fine for now